tests/Kernel/Paragraphs/ParagraphsTestBase.php

```diff
@@ -32,6 +32,7 @@ abstract class ParagraphsTestBase extends AbstractKernelTestBase {
     'options',
     'oe_paragraphs',
     'allowed_formats',
+    'typed_link',
```

Comment:
This should be added in the `SocialMediaFollowTest` and not here.

Reply:
oe_paragraph component will install all the paragraphs and typed_link is required. Tests will fail for the other paragraphs if it's added only in `SocialMediaFollowTest`.

Reply:
The first issue is that we need to set the dependency on the oe_paragraphs branch in composer. Otherwise no social media follow paragraph is available at all.